pkgs/applications/virtualization/virtualbox/default.nix

```diff
@@ -251,7 +251,11 @@ in stdenv.mkDerivation {
 
     mkdir -p "$out/share/virtualbox"
     cp -rv src/VBox/Main/UnattendedTemplates "$out/share/virtualbox"
-    ln -s "${virtualboxGuestAdditionsIso}/VBoxGuestAdditions_${version}.iso" "$out/share/virtualbox/VBoxGuestAdditions.iso"
+    ln -s "${virtualboxGuestAdditionsIso}" "$out/share/virtualbox/VBoxGuestAdditions.iso"
+    if ! [[ -f "$out/share/virtualbox/VBoxGuestAdditions.iso" ]]; then
```

Comment:
Is this really needed?

Reply:
No, it's just a check to avoid similar problems in the future.  If you don't like it, I can drop the second commit.

Reply:
After removing the second commit, I will approve the PR

Reply:
Done.